manager/testdata/new-blueprints/scenario2/transform-config-module.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: M4DModule
+metadata:
+  name: transform-conf
+  namespace: m4d-system
+  labels:
+    version: 0.0.1
+    name: transform-conf
+spec:
+  chart:
+    name: ghcr.io/mesh-for-data/m4d-arrow-flight-transform-conf:0.1.0
+    values: {}
+  type: fybric-action
```

Comment:
What are types used for? Just documentation purposes or do they impact something?

Reply:
@roee88 Isn't this supposed to be 'type: config' to indicate that it's configuring a predefined service or other module?  If I remember correctly you said we have config modules today.  Can link to an example?

Reply:
@froesef I spoke with Roee.  We agreed that an M4DModule.spec.type can be service, config or plugin.  If it's a plugin it will have a pluginType such as wasm, vault, etc.